ui/src/index.js

```diff
@@ -5,4 +5,21 @@ import App from "./App";
 import registerServiceWorker from "./registerServiceWorker";
 
 ReactDOM.render(<App />, document.getElementById("root"));
-registerServiceWorker();
+
```

Comment:
This seems a bit excessive for code comments, most of this context could probably just go in the PR description.

Reply:
Done